src/demo-app/a11y/button-toggle/button-toggle-a11y.html

```diff
@@ -0,0 +1,62 @@
+<section>
+  <h2>Single button toggle</h2>
+  <md-button-toggle>Yes</md-button-toggle>
+</section>
+
+<section>
+  <h2>Button toggles with icons</h2>
+  <md-button-toggle-group name="alignment">
```

Comment:
For single-select groups, they should have an `aria-label` or `aria-labelledby` the same as radio buttons.

For multi-select, I'm just now realizing that it might make sense to put `role="group"` on the group with an aria-label. How does the experience change with that?